### PR TITLE
Driver added

### PR DESCRIPTION
**Pull request by IrinaMakaryan**
new changes added

### CODE REVIEW

File: src/test/java/com/mutual/test/utilities/Driver.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.mutual.test.utilities;
+
+public class Driver {
+    public static void main(String[] args) {
+        System.out.println("Hello");
+        System.out.println("Hi");
```
Comment (avsank) [inline src/test/java/com/mutual/test/utilities/Driver.java:6]: Agree

Comment (avsank) [reply src/test/java/com/mutual/test/utilities/Driver.java:6]: Why not Good evening

**Review by avsank — CHANGES_REQUESTED** [review]
Need explanation.

File: src/test/java/com/mutual/test/utilities/Driver.java (hunk 1 above)
Comment (IrinaMakaryan) [reply src/test/java/com/mutual/test/utilities/Driver.java:6]: thank you

**Review by avsank — APPROVED** [review]
Approved by Anna

**Pull request merged**